config.py

```diff
@@ -2,8 +2,9 @@
 basedir = os.path.abspath(os.path.dirname(__file__))
 
 class Config(object):
-    SECRET_KEY = ''
+    SECRET_KEY = 'test'
     SQLALCHEMY_DATABASE_URI = 'sqlite:///' + os.path.join(basedir, 'redlure.db' + '?check_same_thread=False')
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     CERT_PATH = 'redlure-cert.pem'
     KEY_PATH = 'redlure-key.pem'
+    PASSPHRASE = 'test'
```

Comment:
I'd prefer to leave unset and have operator fill manually if intending to use with docker

```suggestion
    PASSPHRASE = '' # hardcode option for Docker use
```